x/ecocredit/server/basket/msg_create.go

```diff
@@ -36,7 +36,7 @@ func (k Keeper) Create(ctx context.Context, msg *basket.MsgCreate) (*basket.MsgC
 		}
 	}
 
-	err = k.distKeeper.FundCommunityPool(sdkCtx, fee, curator)
+	err = k.bankKeeper.BurnCoins(sdkCtx, ecocredit.ModuleName, fee)
```

Comment:
we need to send the fee to the ecocredit module first before burning: https://github.com/regen-network/regen-ledger/blob/2151ad1f1feb2747c571002ad67c480adaa19c60/x/ecocredit/server/basket/msg_take.go#L57-L65

Reply:
Nice catch. Fixed https://github.com/regen-network/regen-ledger/pull/1167/commits/8cb40073ec7b74798c29a8c2ef759679be45e870